shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -39,6 +39,11 @@ FLUTTER_EXPORT
  */
 @property(nonatomic) FlutterMouseTrackingMode mouseTrackingMode;
 
+/**
+ * This just returns the NSPasteboard so that it can be mocked in the tests.
+ */
+@property(nonatomic, readonly, nonnull) NSPasteboard* _pasteboard;
```

Comment:
This still needs to be moved to the private header.

Also, a property name should not start with `_`; the property and the ivar are distinct.

Reply:
Where exactly should this be moved to?  Sorry for my lack of Objective C knowledge.  I tried moving this inside the .mm file, but then I couldn't mock it from the test.

👍  About the underscore.

Reply:
FlutterViewController_Internal.h

Reply:
Ah thanks, I missed that.

Though it seems to also give me errors when the test tries to mock it.  I just copied this exactly over to the _Internal header.  Is there something else I need to do to make it accessible to the test?

```
FlutterViewControllerTest.mm:32:31: error: no known instance method for selector 'pasteboard'
```

Reply:
Did you include that header in the test file?

Reply:
That was it, thanks.